src/main.js

```diff
@@ -18,12 +18,14 @@ const router = new VueRouter({
 Vue.use(VueRouter)
 Vue.use(VuePageTransition)
 
-new Vue({
+const VueApp = new Vue({
   router,
   store,
   vuetify,
   render: h => h(App)
-}).$mount('#app')
+})
+
+VueApp.$mount('#app')
 
 if (window.Cypress) {
```

Comment:
@vickywane the PR still tries to commit the cypress line. See above. Did you sync your local changes with the remote branch?

Reply:
Yes, my branch is fully synced, 

Previously the vue window was being injected into the app in this conditional statement and that was what i removed. 
E.g `window.vue = VueApp`. 

The current Vue instance is still being used (see [comment](https://github.com/ambianic/ambianic-ui/pull/581#issuecomment-886987283) ). It was there before this PR ( see [main.js](https://github.com/ambianic/ambianic-ui/blob/master/src/main.js) file in master branch )